From 57eaf65accb8d058644dc8eee7332e0ec0c1952f Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Thu, 20 Jul 2017 20:12:10 +0200
Subject: [PATCH] [cmake]: De-duplicate "else" to fix build with cmake-3.9

Otherwise it errors out with:
"CMake Error at kdeui/CMakeLists.txt:316 (else): A duplicate ELSE
command was found inside an IF block."
Also adjust the indentation to match the surrounding lines and remove
the arguments from else() and endif().
---
 kdeui/CMakeLists.txt | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git kdeui/CMakeLists.txt kdeui/CMakeLists.txt
index d6ec8b47e9..cadb52e713 100644
--- kdeui/CMakeLists.txt
+++ kdeui/CMakeLists.txt
@@ -311,11 +311,9 @@ if (Q_WS_MAC AND MAC_USE_OSXKEYCHAIN)
     FIND_LIBRARY(SECURITY_LIBRARY Security)
     set(kdeui_LIB_SRCS ${kdeui_LIB_SRCS} util/kwallet_mac.cpp util/qosxkeychain.cpp)
     add_definitions(-DMAC_USE_OSXKEYCHAIN)
-else(Q_WS_MAC AND MAC_USE_OSXKEYCHAIN)
-     set(kdeui_LIB_SRCS ${kdeui_LIB_SRCS} util/kwallet.cpp)
-else(Q_WS_MAC AND MAC_USE_OSXKEYCHAIN)
-  set(kdeui_LIB_SRCS ${kdeui_LIB_SRCS} util/kwallet.cpp)
-endif(Q_WS_MAC AND MAC_USE_OSXKEYCHAIN)
+else()
+    set(kdeui_LIB_SRCS ${kdeui_LIB_SRCS} util/kwallet.cpp)
+endif()
 
 if(NOT WINCE)
 set(kdeui_LIB_SRCS
